code/modules/admin/fun_balloon.dm

```diff
@@ -83,3 +83,64 @@
 		PoolOrNew(/obj/effect/overlay/temp/sparkle, M)
 		M.forceMove(T)
 		M << "<span class='notice'>Pop!</span>"
+
+/obj/effect/station_crash
+	name = "station crash"
+	desc = "With no survivors!"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "syndballoon"
+	anchored = TRUE
+
+/obj/effect/station_crash/New()
+	for(var/S in SSshuttle.stationary)
+		var/obj/docking_port/stationary/SM = S
+		if(SM.id == "emergency_home")
+			var/new_dir = turn(SM.dir, 180)
+			SM.loc = get_ranged_target_turf(SM, new_dir, rand(3,15))
+			break
+	qdel(src)
+
+
+
+//meteor/asteroid shuttle slamming into the station
+
+/obj/effect/station_crash
+	name = "station crash"
+	desc = "With no survivors!"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "syndballoon"
+	anchored = TRUE
```

Comment:
You appear to have defined this effect twice. And also, since it takes effect immediately, it means that if you load the asteroid template as a preview, or just paste it in, or hypothetically when people can change their mind to what shuttle they want, it'll have permanent effects on the station. This is why I have some of my fun balloons only pop when it's 60 seconds to go on the arrival.